code/edge/src/androidTest/java/com/adobe/marketing/mobile/IdentityStateFunctionalTests.java

```diff
@@ -67,9 +72,15 @@ public void setup() throws Exception {
 		latch.await();
 
 		assertExpectedEvents(false);
+		mockNetworkService.reset();
```

Comment:
Similar to previous comments: 
Add mockNetworkService.reset() to resetTestExpectation?